### PR TITLE
Add the new person detection model (203)

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**
Here the new person detection model (0203). I observe some problems with the accuracy checker which solution is my first priority now.

### CODE REVIEW

File: models/intel/person-detection-0203/accuracy-check.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+models:
+  - name: person-detection-0203
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
```
Comment (eaidova) [inline models/intel/person-detection-0203/accuracy-check.yml:7]: please remove tags and second launcher

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by eaidova** [comment]
@GalyaZalesskaya which kind of issue with AC do you  observe?

**Comment by GalyaZalesskaya** [comment]
I observed the accuracy drop for my models and finally found out the reason for it. It is because of that fact that the `class_agnostic_detection` adapter uses `scale` parameter which is number. It works great with a square resolution, but don't work for a rectangular. So I'll fix the code of the `accuracy_checker` to be able to pass a tuple of scales instead of one number. Do you approve this modification?

**Comment by eaidova** [comment]
> @eaidova I observed the accuracy drop for my models and finally found out the reason for it. It is because of that fact that the `class_agnostic_detection` adapter uses `scale` parameter which is number. It works great with a square resolution, but don't work for a rectangular. So I'll fix the code of the `accuracy_checker` to be able to pass a tuple of scales instead of one number. Do you approve this modification?

Sounds good, but I need to look on it :) I provide more detailed feed back, when you push these changes.

**Comment by GalyaZalesskaya** [comment]
We've made the decision to publish only one model instead  of 3, so I've removed 0203 and 0204 models and renamed 0205 to 0203t to keep the naming logic

File: tools/accuracy_checker/accuracy_checker/adapters/detection.py [hunk 2]
```diff
@@ -356,13 +356,16 @@ def parameters(cls):
         parameters.update({
             'output_blob': StringField(optional=True, default=None, description="Output blob name."),
             'scale': NumberField(optional=True, default=1.0, description="Scale factor for bboxes."),
+            'scales': ListField(optional=True, default=None,
+                                description="List with width and height scale factors for bboxes."),
```
Comment (GalyaZalesskaya) [inline tools/accuracy_checker/accuracy_checker/adapters/detection.py:360]: @eaidova Here are changes I've made to support several scales for bboxes. I wasn't find a way to pass to `scale` parameter both the number and list of 2 numbers, so I added the new  `scales` parameter. Is it appropriate or I missed some way to do it? Feel free to criticize me

Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/detection.py:360]: there is way how it do it.

1. change type of NumberField to BaseField
2. use [get_or_parse_value](https://github.com/openvinotoolkit/open_model_zoo/blob/75ec8ad4b6c4fd7fb161ae9d2c3056281b2443de/tools/accuracy_checker/accuracy_checker/utils.py#L317) function from utils for value parsing. 

As example, you can use normalization preprocessing, which support both per channel mean/std and single value for all channels
https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/accuracy_checker/preprocessor/normalization.py#L55

**Comment by eaidova** [comment]
@GalyaZalesskaya please add new model to [index.md](https://github.com/openvinotoolkit/open_model_zoo/blob/develop/models/intel/index.md)

File: tools/accuracy_checker/accuracy_checker/adapters/detection.py (hunk 2 above)
Comment (GalyaZalesskaya) [reply tools/accuracy_checker/accuracy_checker/adapters/detection.py]: Your solution is quite elegant and great. Thank you for the idea and detailed instructions!

File: models/intel/person-detection-0203/description/person-detection-0203.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+| AP @ [ IoU=0.50:0.95 ]          | 0.412 (internal test set)                 |
+| GFlops                          | 6.519                                     |
+| MParams                         | 2.394                                     |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+## Performance
```
Comment (vladimir-dudnik) [inline models/intel/person-detection-0203/description/person-detection-0203.md:25]: we are removing empty sections from documentation (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)), please do that as well

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

**Review by vladimir-dudnik — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
please add soft link to AC config from old configs locations

**Comment by eaidova** [comment]
@GalyaZalesskaya please rebase your branch, tests failed due to your branch does not contains changes for work with the latest release

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]
do not merge until accuracy validation release criteria met

**Review by vladimir-dudnik — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Pull request merged**